app/lib/shared/configuration.dart

```diff
@@ -253,13 +173,11 @@ class Configuration {
 
   /// Create a configuration based on the environment variables.
   factory Configuration.fromEnv(EnvConfig env) {
-    if (env.gcloudProject == 'dartlang-pub') {
-      return Configuration._prod();
-    } else if (env.gcloudProject == 'dartlang-pub-dev') {
-      return Configuration._dev();
-    } else {
-      throw Exception('Unknown project id: ${env.gcloudProject}');
+    if (File(env.configPath).existsSync()) {
```

Comment:
Let's keep the defaults for `env.gcloudProject == ''`, as in the current method we are using the same `Dockerfile` and setup for both staging and prod. This may change in the future, but let's not change too many things at once.